sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -322,16 +322,22 @@ private static int ParseHealthcareEntityIndex(string reference)
 
         internal static PiiTask ConvertToPiiTask(RecognizePiiEntitiesAction action)
         {
+            var parameters = new PiiTaskParameters()
+            {
+                Domain = action.DomainFilter.GetString() ?? (PiiTaskParametersDomain?)null,
```

Comment:
Just double checking: can `DomainFilter` be null? We need to add a null check if that's the case: ```
```cs
action.DomainFilter?.GetString()
```

Reply:
Good check! Although `DomainFilter` is an non nullable enum so we should be safe